### PR TITLE
MKProj.Minerva version 1.1.1

### PR DESCRIPTION
**Pull request by Mustafif**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34466)

### CODE REVIEW

File: manifests/m/MKProj/Minerva/1.1.1/MKProj.Minerva.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: MKProj.Minerva
+PackageVersion: 1.1.1
+PackageLocale: en-US
+Publisher: MKProj
+PackageName: Minerva
```
Comment (ItzLevvie) [inline manifests/m/MKProj/Minerva/1.1.1/MKProj.Minerva.locale.en-US.yaml:8]:
```suggestion
PackageName: minerva
```

File: manifests/m/MKProj/Minerva/1.1.1/MKProj.Minerva.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: MKProj.Minerva
+PackageVersion: 1.1.1
+PackageLocale: en-US
+Publisher: MKProj
```
Comment (ItzLevvie) [inline manifests/m/MKProj/Minerva/1.1.1/MKProj.Minerva.locale.en-US.yaml:7]:
```suggestion
Publisher: Mustafif
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by Mustafif — COMMENTED** [review]
All good

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Missing these changes: https://github.com/microsoft/winget-pkgs/pull/34466#pullrequestreview-801699268

It must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/34466#pullrequestreview-801812306

**Pull request closed without merging**